docs/guide/config-composer-json.md

```diff
@@ -0,0 +1,144 @@
+## Setting up your composer.json <span id="config-composer-json"></span>
+
+<p align="justify">There are several ways to use the assets in Yii, you can use it in the traditional way in Yii2 by placing the Bower and NPM dependencies, in this case all packages are downloaded from:</p>
+
+- [AssetPackagist](https://asset-packagist.org/).
+
+Directory structure:
+
+- vendor
+  - bower
+  - npm
+
+```    
+{
+    "name": "assets/assets-bootbox",
+    "type": "library",
+    "minimum-stability": "dev",
+    "require": {
+        "php": "^7.2.0",
+        "npm-asset/bootbox": "@dev"
+    },
+    "config": {
+        "process-timeout": 1800,
+        "fxp-asset": {
```

Comment:
That is not required in order to use asset-packagist. It was there in order not to conflict with fxp Composer plugin installed globally.